README.md

```diff
@@ -151,6 +151,31 @@ for more info.
 To run rustfmt after this, use `cargo run --bin rustfmt -- filename`. See the
 notes above on running rustfmt.
 
+All command should be run in the project's root directory.
+
+### Update Toolchain
+
+Make sure to have installed the correct toolchain and all components.
+The needed parts can be found in the file `rust-toolchain`. For example:
```

Comment:
```suggestion
rustfmt depends on compiler internals and therefore pins a specific version of the nightly compiler.
Make sure you've installed the correct nightly [toolchain](https://rust-lang.github.io/rustup/concepts/toolchains.html) and all [components](https://rust-lang.github.io/rustup/concepts/components.html) listed in rustfmt's [`rust-toolchain` file](https://github.com/rust-lang/rustfmt/blob/master/rust-toolchain).
```

It's probably worth mentioning this information before telling users to build rustfmt with `cargo build`